addons/l10n_lt/demo/account_account_template_demo.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo noupdate="1">
+    <!-- Adding Demo Account, because `bank_statement_reconciliation`
+         tour tests expects account with code '100000' exists in COA. -->
+    <record id="account_account_demo" model="account.account.template">
```

Comment:
@oerp-odoo I think this account(converted from template to real account) won't be loaded in a demo database where Lithuanian localization is automatically installed(that is, when you create a db with demo data and set country to Lithuania and install account), because when this demo record is created, `try_loading_for_current_company` would have already been executed. @jco-odoo do you have any idea if we need this demo account ?

Reply:
@rgo-odoo you are right. This won't load. Though I changed it to template, because creating `account.account` can clash if there is generic COA installed and you would try to install/upgrade `l10n_lt` one, because generic one has account with same code.

Now the reasoning why we added this account is that, if some module depends on localization (`l10n_lt`), tests will fail, because it expects account with code `100000`.

I guess I can remove it, but then as mentioned, tests might fail because of that (though on standard odoo, it should pass either way, because nothing depends on `l10n_lt`)

Reply:
yes, you are right. Tests should not depend on account with specific code and should be improved if it fails for this reason. So demo account can be removed from here.